### PR TITLE
docs: add tutorial + improve main doc page

### PR DESCRIPTION
**Pull request by raphael0202**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Open Prices
```
Comment (raphodn) [inline docs/README.md:1]: Maybe you wanted to edit the root README instead ? (and we could delete this docs/README ?)

File: docs/assets/img/proof-examples.png
Comment (teolemon) [inline docs/assets/img/proof-examples.png:1]: What's wrong with the second example ?

Comment (teolemon) [reply docs/assets/img/proof-examples.png:1]: Also, do you have a source template you can commit ?

Comment (raphodn) [reply docs/assets/img/proof-examples.png:1]: Same for the second image 😅
Maybe put a blurred image ? And/or a badly cropped image ?

**Pull request merged**